### PR TITLE
Quick start rewrite

### PR DESCRIPTION
**Pull request by chrisranderson**
For #444.

~~My only question: does the `-g` flag for npm installs require sudo?~~ Checked. According to https://docs.npmjs.com/getting-started/installing-npm-packages-globally it's discouraged.

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -1,14 +1,48 @@
 Quick start
 ===========
 
-Install using `nodejs <http://nodejs.org>`_::
+Installation 
+------------
+
+First, install `git <https://git-scm.com/downloads>`_. 
+
+Second, install `Node.js <http://nodejs.org>`_. Webppl is written in JavaScript, and requires Node to run. After it is installed, you can use npm (**n**\ ode **p**\ ackage **m**\ anager) to install webppl::
```
Comment (null-a) [inline docs/quickstart.rst]: I *think* "WebPPL" is used more often in the docs at the moment, so I'd use that for now. It also means we can punt on whether or not "webppl" should be "Webppl" at the beginning of a sentence...

File: docs/quickstart.rst [hunk 2]
```diff
@@ -1,14 +1,48 @@
...
+    return phrase
+
+Run it with this command::
+
+    webppl test.wppl
+
+Learning
+--------
+
+If you're new to probabilistic programming, `Probabilistic Models of Cognition <https://probmods.org/>`_ is a great place to start learning the paradigm. It uses Church for demonstrations, but the principles will apply when you are using webppl.
```
Comment (null-a) [inline docs/quickstart.rst]: We maybe shouldn't assume people will know what Church is. Maybe we just say "uses the language Church..." or something.

File: docs/quickstart.rst [hunk 3]
```diff
@@ -1,14 +1,48 @@
...
+        return flip(.5) ? "Hello" : "Howdy"
+    }
+    
+    var audience = function () {
+        return flip(.5) ? "World" : "Universe"
+    }
+    
+    var phrase = greeting() + ", " + audience() + "!"
+    
+    return phrase
```
Comment (null-a) [inline docs/quickstart.rst]: As you probably know, we automatically `return` the last expression in the program. I wonder if we should introduce that idea here by dropping the `phrase` variable?

**Comment by null-a** [comment]
Looks good! I had a quick pass through and added a few comments inline.

File: docs/quickstart.rst (hunk 1 above)
Comment (chrisranderson) [inline docs/quickstart.rst]: Read my mind! I was wondering...

I'll change that.

File: docs/quickstart.rst (hunk 3 above)
Comment (chrisranderson) [inline docs/quickstart.rst]: I would count that as distracting, but I'm okay with changing it. It is kinda weird coming from JS land.

Comment (chrisranderson) [inline docs/quickstart.rst]: On the other hand, it would be nice to let them know "this isn't just JS".

File: docs/quickstart.rst (hunk 2 above)
Comment (chrisranderson) [inline docs/quickstart.rst]: What if I add a link to http://projects.csail.mit.edu/church/wiki/Church?

**Comment by stuhlmueller** [comment]
Yes, looks good. Ready to merge once `[W]ebppl` has been replaced with `WebPPL` throughout.

**Comment by chrisranderson** [comment]
Eh, sorry. I'm getting confused with all of my branches it looks like.

**Comment by stuhlmueller** [comment]
I assume you've seen it already, so just in case: there is a [leftover bit](https://github.com/probmods/webppl/pull/559/files#diff-9d8a6f5b32d4a70a56734fd0eb50dd5bR41) from the merge.

**Comment by chrisranderson** [comment]
@stuhlmueller  my git incompetence shows through. Should be good now.

**Comment by stuhlmueller** [comment]
Thanks! There's still a small issue: `<<<<<<< HEAD` shouldn't be in `quickstart.rst`.

**Comment by chrisranderson** [comment]
Sorry :/ fixed.

**Comment by stuhlmueller** [comment]
Thanks!

**Pull request merged**